t/app/controller/waste_merton.t

```diff
@@ -344,10 +344,23 @@ FixMyStreet::override_config {
         is $report->category, 'Request new container';
         is $report->title, 'Request new Recycling Blue Stripe Bag';
     };
-    subtest 'Weekly collection cannot request a blue stripe bag' => sub {
+    subtest 'Above-shop address' => sub {
         $e->mock('GetServiceUnitsForObject', sub { $above_shop_data });
         $mech->get_ok('/waste/12345/request');
```

Comment:
Have added some basic tests.
This took me far too long because I hadn't noticed this bit was fetching '/waste/12345/request' and not ''/waste/12345' and I kept wondering why I wasn't getting the expected output 🤦‍♀️